### DISCUSSION TITLE
splitChunks is not working for speciifc module.

### DISCUSSION THREAD

**Discussion by Surajbisht15** [webpack/webpack#17894] [Q&A]
I want to create chunks for app-routing.module.ts; here is my optimization code:

`optimization: {
        splitChunks: {
            cacheGroups: {
                routingChunk: {
                    test: /src\/app\/app-routing\.module\.ts/,
                    name: 'routingChunk',
                    chunks: 'initial'
                }
            }
        }
    }`

app-routing.module.ts is inside **`src/app/app-routing.module.ts`**;

What am I doing wrong here ?

**Comment by alexander-akait** [comment]
Please provide reproducible example, it is very hard to help without real configuration/code, thank you
